### ISSUE TITLE
Im sorry, But I'm so confused. Nothing will compile, or even load.

### ISSUE THREAD

**Issue by HoggieBoggie** [Jessecar96/SteamBot#367]
I know you guys don't like these kinds of things.. but hear me out.

I downloaded the clone yesterday through newly installed github's gitshell. I opened the solution file in Monodevelop , and compiled it fine. Ran it, connected to my account and everything, all was fine.
About 10min after successfully compiling it the first time, I went back to work on a user handle. All I did was change the AdminUserHandle very slightly, all I did was make the trade return true if non-admin's traded. I compiled it, and got the error explained in the Installation guide about "ProtoBuf". I built the solution twice, three times, it would NOT compile at all! I didn't change anything else, except for one thing!
So I accepted my defeat and deleted the SteamBot folder, and just re-cloned it, and tried again. Nope. Still wouldn't work, this time all the code was exactly the same too.
Alright, fine. Ill try MSVS10. It wouldn't even load into it. It saying it can't read "SteamKit2.csproj". I tried this a million times.
Maybe its just my computer? So I tried it on a separate computer, clean install Windows 7. Installed all the SDK's and everything. Open the solution in Mono, nope, won't compile. Open in into WSVS10, nope won't compile. Open it into MSVS12, still no.
Through out this period of time I've downloaded a new clone at least 10 times.

I'm sorry for this wall of text, I just have NO idea whats going on.
Any help would be LOVED.

At this point I think I might just try to code my own, I just didn't want to re-invent the wheel, so I figured giving this a try.

tl dr- Won't load, nor compile.

Thanks.
